WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -8298,7 +8296,7 @@
 	430.1.1 = {
 		add_spouse = 1003024
 	}
-	470.1.1 = {
+	475.1.1 = {
```

Comment:
What was the justification for this?

Reply:
His daughter was born in 475

Reply:
His daughter was born in 475.

Reply:
Isn't that daughter a fictional character?

Reply:
Yes, that daughter is fictional character. At all, she looks very weird : being born when mother was 65 years old and being wife of her own nephew in same age. I don't wanted to made Anastasia older than husband, so i pushed death date of Pompeius to her birth date.

Reply:
Is the mother being 65 kept or not? Because that can't be intended?

Reply:
IMO this is the best way to fix this. If i change Anastasia's age, she will become elder than husband.

Reply:
But a woman at the age of 65 can impossibly give birth.

Reply:
@Symmachus I've checked again and that daughter Anastasia is most likely a historical character: https://www.geni.com/people/Anastasia/6000000006101423446
I agree with @loup99 that her mother's age on birth is implausible. We should probably just make Anastasia's birth date earlier.

Reply:
> 
> 
> @Symmachus I've checked again and that daughter Anastasia is most likely a historical character: https://www.geni.com/people/Anastasia/6000000006101423446
> I agree with @loup99 that her mother's age on birth is implausible. We should probably just make Anastasia's birth date earlier.
> And I don't know what husband you have in mind, 1003042 has no husband.

I don't want to make Anastasia a lot elder than husband (Flavius Pompeius is her husband - they have a son). I can make her older on 5 or 10 years, but this is not completely solve the problem.

Reply:
Revert this change and make Anastasia born 15 or so years earlier?

Reply:
I've added the missing marriage link.